pkg/asset/ignition/bootstrap/bootstrap.go

```diff
@@ -292,10 +292,11 @@ func (a *Bootstrap) addStorageFiles(base string, uri string, templateData *boots
 	}
 
 	filename := path.Base(uri)
+	parentDir := path.Base(path.Dir(uri))
 
 	var mode int
 	appendToFile := false
-	if path.Base(path.Dir(uri)) == "bin" {
+	if parentDir == "bin" || parentDir == "dispatcher.d" {
```

Comment:
Also it would be nice if you can include why these files need to have same permissions as bin files?

Reply:
Yeah, a comment would probably be nice.  It's a directory of scripts executed by NetworkManager.